README.md

```diff
@@ -13,9 +13,13 @@
 * [OpenStack](docs/user/openstack/README.md)
 * [OpenStack (UPI)](docs/user/openstack/install_upi.md)
 * [Power](docs/user/power/install_upi.md)
+* [Power Virtual Server](https://docs.openshift.com/container-platform/latest/installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.html)
```

Comment:
This is the IPI variant of the "Power" platform listed above, also I think it is more commonly referr to as PowerVS, so to be consistent with the other platforms, I would change it like so:

* [PowerVS](https://docs.openshift.com/container-platform/latest/installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.html)
* [PowerVS (UPI)](docs/user/power/install_upi.md)